namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-preprod/resources/s3.tf

```diff
@@ -10,11 +10,46 @@ module "analytical_platform_s3_bucket" {
   infrastructure-support = var.infrastructure_support
   namespace              = var.namespace
 
+  bucket_policy = data.aws_iam_policy_document.bucket-policy.json
+
   providers = {
     aws = aws.london
   }
 }
 
+locals {
+  bucket_arn = "arn:aws:s3:::cloud-platform-1ed50d9e895c5fd29329796e35c3cc10"
+}
+
+data "aws_iam_policy_document" "bucket-policy" {
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = [module.analytical-platform.aws_iam_role_arn]
+    }
+    actions = [
+      "s3:ListBucket",
+    ]
+    resources = [
+      local.bucket_arn
+    ]
+  }
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = [module.analytical-platform.aws_iam_role_arn]
+    }
+    actions = [
+      "s3:GetObject",
+      "s3:GetObjectAcl",
+      "s3:ListObjectsV2",
```

Comment:
@Mjwillis , can u pls remove the `"s3:ListObjectsV2",` from the bucket policy. It is not a valid action for the bucket itself.